src/pybind/mgr/restful/common.py

```diff
@@ -6,7 +6,7 @@
 
 # Implemented osd commands
 OSD_IMPLEMENTED_COMMANDS = [
-    'scrub', 'deep-scrub', 'repair'
+    'scrub', 'deep-scrub', 'deep_scrub', 'repair'
```

Comment:
Why do we need to expose `deep_scrub` to the restful module? It was never exposed in earlier versions https://github.com/ceph/ceph/blob/quincy/src/pybind/mgr/restful/common.py#L9 and https://github.com/ceph/ceph/blob/reef/src/pybind/mgr/restful/common.py#L9?